core_v2/sources/domain_e2e_tests.move

```diff
@@ -597,13 +917,7 @@ module aptos_names_v2::domain_e2e_tests {
         rando: signer,
         foundation: signer
     ) {
-        // Setup test env for v1
-        let users = aptos_names::test_helper::e2e_test_setup(aptos_names, user, &aptos, rando, &foundation);
-
-        // Setup test env for v2
-        account::create_account_for_test(@aptos_names_v2);
-        aptos_names_v2::domains::init_module_for_test(aptos_names_v2);
-        aptos_names_v2::config::set_fund_destination_address_test_only(signer::address_of(&foundation));
```

Comment:
since we are calling `set_fund_destination_address_test_only()` in `e2e_test_setup()` below, does that mean we are actually duplicating the operation here?

Reply:
i see thanks for explanation